README.md

```diff
@@ -1 +1,25 @@
-# squirrel_prediction
+# squirrel_prediction repository
+
+
+Repository for prediction and learning related SQUIRREL packages.
+
+Installation requiremets
+```
+sudo apt-get install scipy
```

Comment:
@ipa-nhg can this be inserted to package.xml so that rosdep install covers it? Currently, scipy is not part of https://github.com/ros/rosdistro/blob/master/rosdep/base.yaml